client/components/jetpack/backup-getting-started/style.scss

```diff
@@ -0,0 +1,93 @@
+$card-height-desktop: 250px;
+$preview-max-width: 360px;
+$content-max-width: 210px;
+
+.backup-getting-started {
+	display: grid;
+	height: $card-height-desktop;
+	background-image: url( 'calypso/assets/images/jetpack/getting-started-backup-bg.svg' );
+	background-color: #f9f9f6;
+	grid-template-columns: minmax( 0, $preview-max-width ) minmax( $content-max-width, 1fr );
+	grid-column-gap: 1.5rem;
+
+	@include breakpoint-deprecated( '<660px' ) {
```

Comment:
You may be able to use the [Gutenberg breakpoints](https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/base-styles/_breakpoints.scss) here (with the `break-medium` or `break-small` mixins), though they're not exactly the same value.

Reply:
Thanks!